src/file.cr

```diff
@@ -150,8 +151,8 @@ class File < IO::FileDescriptor
   # File.write("foo", "foo")
   # File.executable?("foo") # => false
   # ```
-  def self.executable?(path) : Bool
-    Crystal::System::File.executable?(path)
+  def self.executable?(path : Path | String) : Bool
+    Crystal::System::File.executable?(path.to_s)
```

Comment:
would it make more sense for the `Crystal::System::File` methods to take a String or a Path instead of calling `to_s` on all these?

Reply:
This would just move the `to_s` to `Crystal::System::File`. I think it is fine for those implementation methods to accept only `String`.

Reply:
That makes sense :)